lib/3scale/backend/storage_helpers.rb

```diff
@@ -241,6 +246,38 @@ def cfg_sentinels_handler(options)
             options
           end
```

Comment:
Should we make sure that here sentinels are not empty? 

Up there we have `next if sentinel.empty?` and then `end.compact` so basically we can end up with an empty `options[:sentinels]`

Reply:
Are we not already doing it?

https://github.com/3scale/apisonator/blob/5125eed61e7f71be67b54712aab7f88784876ba1/lib/3scale/backend/storage_helpers.rb#L219

Reply:
I saw that but it is before the `compat` call. So we may still end up with empty sentinels list.

Reply:
You're right, I made some changes here: https://github.com/3scale/apisonator/pull/384/commits/9f85a48c3cd64c94f631f0660f5fccac99d45c7e